contracts/governance/src/ZeroExVotes.sol

```diff
@@ -72,15 +72,15 @@ contract ZeroExVotes is IZeroExVotes, Initializable, OwnableUpgradeable, UUPSUpg
      */
     function getVotes(address account) public view returns (uint256) {
         uint256 pos = _checkpoints[account].length;
-        return pos == 0 ? 0 : _checkpoints[account][pos - 1].votes;
+        return pos == 0 ? 0 : _unsafeAccess(_checkpoints[account], pos - 1).votes;
```

Comment:
Did we determine that solc is smart enough to elide the underflow check here?

Reply:
Ah so we're saving the array bounds check but still doing the underflow check. Wrapping these in `unchecked` did save 24 gas in addition to the 118 gas saved by the array bounds check in `getQuadraticVotes` and `getVotes`.